tests/dofs/dof_tools_04a.cc

```diff
@@ -0,0 +1,120 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (C) 2003 - 2017 by the deal.II authors
+//
+// This file is part of the deal.II library.
+//
+// The deal.II library is free software; you can use it, redistribute
+// it, and/or modify it under the terms of the GNU Lesser General
+// Public License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+// The full text of the license can be found in the file LICENSE at
+// the top level of the deal.II distribution.
+//
+// ---------------------------------------------------------------------
+
+
+// The same as dof_tools_04 but for a parallel::distributed::parallel::distributed::Triangulation
+// instead of a parallel::distributed::Triangulation:
```

Comment:
did you want to say `::Triangulation` here, instead of the namespace-qualified version?